ui/app/services/auth.js

```diff
@@ -441,4 +441,22 @@ export default Service.extend({
       backend: BACKENDS.findBy('type', backend),
     });
   }),
+
+  getOktaNumberChallengeAnswer(nonce) {
+    let namespace = 'undefined';
+    const url = `/v1/auth/okta/verify/${nonce}`;
```

Comment:
Need to check with the docs, that `okta` in this path is a static string, not the auth mount path

Reply:
How do I know if the API requires the auth mount path? The documentation from where I used the api states 
`This documentation assumes the Okta method is mounted at the /auth/okta path in Vault. Since it is possible to enable auth methods at any location, please update your API calls accordingly.`

Reply:
Excellent, that's very relevant documentation! So, if you set up another okta auth method you'll need to name it something other than `okta`-- for example, `okta-2`. Essentially, we just need to add `data.backend` to the params when we call `getOktaNumberChallengeAnswer` because that is the value of where the auth mount is mounted, and then use that value rather than hardcoded `okta` in the path. 
Let me know if that makes sense, or if you'd like more clarification! It would probably make more sense on a pairing walkthrough